sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -2538,6 +2538,78 @@ case class Encode(value: Expression, charset: Expression)
     newLeft: Expression, newRight: Expression): Encode = copy(value = newLeft, charset = newRight)
 }
 
+/**
+ * Converts the input expression to a binary value based on the supplied format.
+ */
+// scalastyle:off line.size.limit
+@ExpressionDescription(
+  usage = """
+            |_FUNC_(str[, fmt]) - Converts the input `str` to a binary value based on the supplied `fmt`.
+            |  `fmt` can be a case-insensitive string literal of "hex", "utf-8", "base2", or "base64".
+            |  By default, the binary format for conversion is "hex" if `fmt` is omitted.
+            |
+            |The function returns NULL if at least one of the input parameters is NULL.
+          """,
```

Comment:
I think we shouldn't use `|`s here. Should be something like:

```
  usage = """
    _FUNC_(expr[, relativeSD]) - Returns the estimated cardinality by HyperLogLog++.
      `relativeSD` defines the maximum relative standard deviation allowed.""",
```

(see the docs of `ExpressionDescription`)

Reply:
Updated, thanks!